participant/src/round2.rs

```diff
@@ -0,0 +1,135 @@
+use crate::Logger;
+use frost::{
+    keys::KeyPackage,
+    round1::{NonceCommitment, SigningCommitments, SigningNonces},
+    round2::{self, SignatureShare},
+    Error, Identifier, SigningPackage,
+};
+use frost_ed25519 as frost;
+use hex::FromHex;
+use std::io::BufRead;
+
+// #[derive(Debug)]
+pub struct Round2Config {
+    pub message: Vec<u8>,
+    pub signer_commitments: Vec<SigningCommitments>,
+}
+
+// TODO: refactor to generate config
+// TODO: handle errors
+pub fn round_2_request_inputs(
+    signing_commitments: SigningCommitments,
+    input: &mut impl BufRead,
+    logger: &mut dyn Logger,
+) -> Result<Round2Config, Error> {
+    logger.log("=== Round 2 ===".to_string());
+
+    logger.log("Number of signers:".to_string());
+
+    let mut signers_input = String::new();
+
+    input.read_line(&mut signers_input).unwrap();
+
+    let signers = signers_input.trim().parse::<u16>().unwrap();
+
+    logger.log("You will receive a message from the coordinator, please enter here:".to_string());
+
+    let mut message_input = String::new();
+
+    input.read_line(&mut message_input).unwrap();
+
+    let message = hex::decode(message_input.trim()).unwrap();
+
+    let mut commitments = vec![signing_commitments];
+
+    for i in 2..=signers {
+        logger.log("Identifier:".to_string());
+
+        let mut identifier_input = String::new();
+
+        input.read_line(&mut identifier_input).unwrap();
+
+        let id_value = identifier_input.trim().parse::<u16>().unwrap();
+        let identifier = Identifier::try_from(id_value).unwrap();
+
+        logger.log(format!("Hiding commitment {}:", i));
```

Comment:
Using `i` here is a bit confusing, my suggestion is to use the `id_value` that was just read

Reply:
Done in [679ac65](https://github.com/ZcashFoundation/frost-zcash-demo/pull/44/commits/679ac65694cde77d9786af3790bbfeb1d714c8e5)